### ISSUE TITLE
/tokens action is missing ?

### ISSUE THREAD

**Issue by ddragosd** [adobe/adobeio-cna-lib-auth-ims#2]
### Expected Behaviour
The code references a `tokens` action. See https://github.com/adobe/adobeio-cna-lib-auth-ims/blob/master/src/providers/IMSAuthCode.js#L51 .

### Actual Behaviour
Should it be included ?

**Comment by Himavanth** [comment]
Yes the tokens action is coming from https://github.com/adobe/adobeio-cna-actions-auth/blob/master/Makefile#L60
Just a small enhancement to your work. 😄

**Comment by ddragosd** [comment]
@Himavanth should we add the instructions in the README in this repo ?
